spec/modules/payloads_spec.rb

```diff
@@ -758,6 +758,14 @@
                           reference_name: 'cmd/unix/pingback_reverse'
   end
 
+  context 'cmd/unix/python' do
+    it_should_behave_like 'payload is not cached',
+                          ancestor_reference_names: [
+                            'adapters/cmd/unix/python'
+                          ],
+                          reference_name: 'cmd/unix/python'
```

Comment:
Given everything else here has a `dynamic_size:` field it might be a good idea to add this in here, even if we are setting it to `false` just to fall in line with existing code? Also does this require `modules_pathname`?